src/apps/components/run.js

```diff
@@ -77,22 +86,27 @@ export class RunApp extends React.Component {
         this.props.dispatch(actions.newError(err));
     };
 
-    runView = () => {
+    runView = (options) => {
         let { dispatch } = this.props;
+        options = options || {};
 
         this.inputs.getValues()
         .then((values) => {
-            return this.view.run(this.props.bundle, values);
+            return this.view.run(this.props.bundle, values, options.debug);
```

Comment:
The change would be backwards incompatible so I'd prefer doing it sooner, but this is fine for now if this code is expected to run against `juttle-client-library@0.7.0` which has the signature used here.